src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,30 +1,70 @@
 package org.usfirst.frc4904.robot;
 
+import org.usfirst.frc4904.standard.custom.sensors.PDP;
+import org.usfirst.frc4904.standard.subsystems.chassis.SolenoidShifters;
+import org.usfirst.frc4904.standard.subsystems.chassis.TankDriveShifting;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.AccelerationCap;
+import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.EnableableModifier;
+import edu.wpi.first.wpilibj.VictorSP;
 
 public class RobotMap {
-	public static class Port {
+	public static class Port { //TODO: Correct Ports
 		public static class HumanInput {
 			public static final int joystick = 0;
 			public static final int xboxController = 1;
 		}
 
 		public static class CANMotor {}
 
-		public static class PWM {}
+		public static class PWM {
+			public static final int leftDriveA = 1;
+			public static final int leftDriveB = 2;
+			public static final int rightDriveA = 3;
+			public static final int rightDriveB = 4;
```

Comment:
Set these to default values for right now (`-1`)

Reply:
fixed